ajax/libs/jquery.sumoselect/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jquery.sumoselect",
+  "version": "3.0.2",
+  "description": "A jQuery plugin that progressively enhances an HTML Select Box into a Single/Multiple option dropdown list",
+  "filename": "jquery.sumoselect.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/HemantNegi/jquery.sumoselect.git"
+  },
+  "keywords": [
+    "multiselect",
+    "dropdown"
+  ],
```

Comment:
I think `https://hemantnegi.github.io/jquery.sumoselect/sumoselect_demo.html` can be set as `homepage`.
What do you think?